Mathlib/Logic/Basic.lean

```diff
@@ -1032,17 +1036,22 @@ theorem BEx.intro (a : α) (h₁ : p a) (h₂ : P a h₁) : ∃ (x : _) (h : p x
   ⟨a, h₁, h₂⟩
 #align bex.intro BEx.intro
 
-theorem ball_congr (H : ∀ x h, P x h ↔ Q x h) : (∀ x h, P x h) ↔ ∀ x h, Q x h :=
+theorem forall_mem_congr (H : ∀ x h, P x h ↔ Q x h) : (∀ x h, P x h) ↔ ∀ x h, Q x h :=
   forall_congr' fun x ↦ forall_congr' (H x)
-#align ball_congr ball_congr
+#align ball_congr forall_mem_congr
 
-theorem bex_congr (H : ∀ x h, P x h ↔ Q x h) : (∃ x h, P x h) ↔ ∃ x h, Q x h :=
+theorem exists_mem_congr (H : ∀ x h, P x h ↔ Q x h) : (∃ x h, P x h) ↔ ∃ x h, Q x h :=
```

Comment:
```suggestion
theorem exists₂_congr (H : ∀ x h, P x h ↔ Q x h) : (∃ x h, P x h) ↔ ∃ x h, Q x h :=
```

Reply:
This already exists, with a different signature: https://leanprover-community.github.io/mathlib4_docs/Init/PropLemmas.html#exists%E2%82%82_congr
Replacing them wasn't obvious to me at this hour.

Reply:
I know 😉 The lemma here is a special case of the existing `exists₂_congr`